tools/serverpod_cli/bin/serverpod_cli.dart

```diff
@@ -38,6 +40,8 @@ final runModes = <String>['development', 'staging', 'production'];
 final Analytics _analytics = Analytics();
 
 void main(List<String> args) async {
+  logger = StdOutLogger();
```

Comment:
This solution has a critical problem.

When running tests there will be no logger initialised.

Therefore we can't rely on a s final singleton initialised in the bin file.

Reply:
Isn't this what the `setUp` method in the test is for?